dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/ServiceRegistryService.java

```diff
@@ -48,22 +57,20 @@ public boolean equals(Object o) {
 
         ServiceRegistryService that = (ServiceRegistryService) o;
 
-        if (!uri.equals(that.uri)) return false;
+        if (!uris.equals(that.uris)) return false;
```

Comment:
Do we care about the orders of `URIs` here? If not, then `list.equals` will fail if both have the same elements but in a different order.

Reply:
I don't think the order should matter.

Reply:
Maybe use `Set` instead of `List`? I don't think it makes any difference in practice, and this is internal-only so we could change later.